app/org/catalyst/slackservice/util/ResultHelper.java

```diff
@@ -16,6 +16,12 @@
                 "error", messages.get(error)))));
     }
 
+    public static CompletionStage<Result> badRequest(String error) {
+        return CompletableFuture.completedFuture(Controller.badRequest( Json.toJson(Map.of(
+                "ok", Boolean.valueOf(false),
+                "error", error))));
+    }
```

Comment:
The logic of the two `badRequest` functions are basically the same. Would it make sense to refactor the original `badRequest` function to be utilize the new `return badRequest(messages.get(error))`?

Reply:
Also ln 61 of `AuthController` has a similar structure.

Reply:
The first method is being used in other classes so I created a second

Reply:
To clarify my original message this is what I was trying to suggest.
```
   public static CompletionStage<Result> badRequest(MessageHandler messages, String error) {
        return badRequest(messages.get(error));
    }

    public static CompletionStage<Result> badRequest(String error) {
        return CompletableFuture.completedFuture(Controller.badRequest(Json.toJson(Map.of(
                "ok", Boolean.valueOf(false),
                "error", error))));
    }
```

Reply:
ah now I see what you mean. yup, that makes sense. will make the change